lib/src/server.dart

```diff
@@ -15,6 +15,9 @@ import 'exception.dart';
 import 'parameters.dart';
 import 'utils.dart';
 
+/// A callback for unhandled exceptions.
+typedef ErrorCallback = void Function(dynamic error, dynamic stackTrace);
```

Comment:
I'm not sure it's worth the `typedef` here. `void Function(dynamic, dynamic)` isn't that long.

Reply:
I prefer keeping the `typedef` since it allows for documentation and naming of the parameters.  This callback is also repeated across several functions in peer.dart and server.dart.

If that really doesn't fit with the style here, I can certainly take it out - let me know.